config/locales/en.yml

```diff
@@ -837,16 +838,19 @@ en:
           h_branding_not_paid: "När detta är gjort behöver du göra färdigt din företagssida och betala din H-märkningsvagift. Mer information om hur du gör det kommer i ett mail efter att du betalt din medlemsavgift."
```

Comment:
Translation, if you want:
When this is done you will need to finish your company page and pay your H-branding fee. More information about how to do that will come after you paid your membership fee.